import/index_scripts/keyWordChainBag.bsh

```diff
@@ -0,0 +1,146 @@
+import org.marc4j.marc.Record;
+import org.marc4j.marc.ControlField;
+import org.marc4j.marc.DataField;
+import org.marc4j.marc.Subfield;
+import java.util.*;
+
+/**
+ * Creates unordered list consisting of a bag of keywords found in the keyword chains
+ */
+
+
+
+final static String KEYWORD_DELIMITER = "/";
+
+public Set/*<String>*/ getKeyWordChainBag(Record record, String fieldSpec) {
+
+	//System.out.println("ENTERING getKeyWordChainBag");
+
+	//System.out.println("FieldSpec " + fieldSpec);
+
+	List/*<DataField>*/ dataFields = record.getDataFields();
+	Map/*<Character, List<String>>*/ keyWordChains = new HashMap/*<Character, List<String>>*/();
```

Comment:
Please rename this to a more descriptive variable name, e.g. "chain_ids_to_chains_map".